pkg/search/proxy/store/store.go

```diff
@@ -191,6 +191,11 @@ func (s *store) Count(string) (int64, error) {
 	return 0, fmt.Errorf("count is not supported in proxy store")
 }
 
+// RequestWatchProgress implements storage.Interface.
+func (s *store) RequestWatchProgress(context.Context) error {
+	return fmt.Errorf("not implemented")
+}
+
```

Comment:
@XiShanYongYe-Chang @ikaven1024 @xigang 
Can you help to confirm if it OK to leave this interface unimplemented?
This interface was introduced by https://github.com/kubernetes/kubernetes/pull/119207.

Reply:
By examining the calling path of the code, confirm that the current return of error has no impact. 

Test in the environment, and verify that the search function is working properly.

Reply:
Thanks.